website/static/js/osfHelpers.js

```diff
@@ -456,14 +457,19 @@ var applyBindings = function(viewModel, selector) {
         throw "Can't bind ViewModel to multiple elements."; // jshint ignore: line
     }
     // Ensure that the bound element is shown
-    if ($elem.hasClass('scripted')){
+    if ($elem.hasClass('scripted')) {
         $elem.show();
     }
     // Also show any child elements that have the scripted class
-    $(cssSelector + ' .scripted').each(function(elm) {
+    $(cssSelector + ' .scripted').each(function (elm) {
         $(this).show();
     });
-    ko.applyBindings(viewModel, $elem[0]);
+
+    //Check if bindings have been applied to the element already
+    var isBound = !!ko.dataFor(elem);
+    if(!isBound) {
```

Comment:
I would prefer this not be included - the warning that you are attempting to bind an object twice is useful in that it tells the developer that something is wrong. I'm concerned that allowing that case to pass silently could complicate debugging.